src/sota_tools/garage_deploy.cc

```diff
@@ -91,18 +98,23 @@ int main(int argc, char **argv) {
     OSTreeHash commit(OSTreeHash::Parse(ostree_commit));
     // Since the fetches happen on a single thread in OSTreeHttpRepo, there
     // isn't really any reason to upload in parallel
-    if (!UploadToTreehub(src_repo, push_credentials, commit, cacerts, false, 1)) {
+    if (!UploadToTreehub(src_repo, push_credentials, commit, cacerts, dry_run, 1)) {
       LOG_FATAL << "Upload to treehub failed";
       return EXIT_FAILURE;
     }
 
-    if (push_credentials.CanSignOffline()) {
-      bool ok = OfflineSignRepo(ServerCredentials(push_credentials.GetPathOnDisk()), name, commit, hardwareids);
-      return static_cast<int>(!ok);
-    }
-    LOG_FATAL << "Online signing with garage-deploy is currently unsupported";
-    return EXIT_FAILURE;
+    return EXIT_SUCCESS;
```

Comment:
Was this used for debugging? The code from line 108 looks unreachable.

Reply:
Crap, that was supposed to be removed. Fixed.